irohad/ordering/impl/on_demand_ordering_service_impl.hpp

```diff
@@ -8,23 +8,100 @@
 
 #include "ordering/on_demand_ordering_service.hpp"
 
+#include <queue>
+#include <shared_mutex>
+#include <unordered_map>
+
+#include <tbb/concurrent_queue.h>
+
+#include "logger/logger.hpp"
+
 namespace iroha {
   namespace ordering {
     class OnDemandOrderingServiceImpl : public OnDemandOrderingService {
      public:
-      // --------------------- | OnDemandOrderingService | ---------------------
+      OnDemandOrderingServiceImpl(
+          size_t transaction_limit,
+          size_t number_of_proposals = 3,
```

Comment:
Please add a description for default values to make the unfamiliar readers more comfortable.